test/terra/noise/test_standard_errors.py

```diff
@@ -71,56 +73,29 @@ def test_mixed_unitary_error(self):
         unitaries = [np.eye(2), np.diag([1, -1])]
         probs = [0.7, 0.3]
         error = mixed_unitary_error([(unitaries[0], probs[0]),
-                                     (unitaries[1], probs[1])],
-                                    standard_gates=True)
-        (op0, p0) = error.error_term(0)
-        (op1, p1) = error.error_term(1)
-        self.assertEqual(op0[0], {"name": "z", "qubits": [0]})
-        self.assertEqual(op1[0], {"name": "id", "qubits": [0]})
-        self.assertEqual(p0, 0.3)
-        self.assertEqual(p1, 0.7)
+                                     (unitaries[1], probs[1])])
+        for i in [0, 1]:
+            op, p = error.error_term(i)
+            self.assertIn(p, probs)
+            expected = unitaries[0] if p == 0.7 else unitaries[1]
+            self.assertTrue(np.allclose(op[0][0].to_matrix(), expected))
```

Comment:
These tests may be clearer if they're more like `self.assertEqual(p, probs[i])` and `np.allclose(op[0][0].to_matrix(), unitaries[i])`.

Reply:
Done d0ffafd

Reply:
Done d0ffafd